front/app/containers/PlatformFooter/index.tsx

```diff
@@ -271,6 +273,27 @@ const PlatformFooter = ({ className }: Props) => {
     name: 'remove_vendor_branding',
   });
 
+  const getLocalizedLogoName = () => {
+    switch (locale) {
+      case 'es-CL':
+      case 'es-ES':
+      case 'ca-ES':
```

Comment:
@amanda-anderson Are we sure we want to show the Spanish version for Catalan? It could make sense, but it could also be controversial.

Reply:
Ah good point - maybe we should check with Isa? If this isn't ok, we'll need another separate PNG/translation for Catalan then :thinking:

Reply:
@amanda-anderson  Sure, I can ask on Slack. But I was wondering if this isn't a more general question about translations and on which locale to fall back for Catalan (for example in admin HQ)?